core/metacling/src/TCling.cxx

```diff
@@ -2026,13 +2035,12 @@ void TCling::RegisterModule(const char* modulename,
       // specifying the relevant include paths we should try loading the
       // modulemap next to the library location.
       clang::Preprocessor &PP = TheSema.getPreprocessor();
-      // Can be nullptr in case of libCore.
-      if (dyLibName)
-         RegisterPrebuiltModulePath(PP, llvm::sys::path::parent_path(dyLibName));
+      RegisterPrebuiltModulePath(llvm::sys::path::parent_path(dyLibName),
+                                 ((isACLiC) ? ModuleName : std::string("module")) + ".modulemap");
```

Comment:
Could you create an local variable for this, with an informative name? E.g. `ModuleMapName`?

Reply:
I was explicitly avoiding so, but I agree it may make the code more readable.